packages/server/graphql/mutations/archiveTimelineEvent.ts

```diff
@@ -0,0 +1,71 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {GQLContext} from '../graphql'
+import ArchiveTimelineEventPayload from '../types/ArchiveTimelineEventPayload'
+import publish from '../../utils/publish'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import TimelineEvent from '../../database/types/TimelineEvent'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import standardError from '../../utils/standardError'
+
+const archiveTimelineEvent = {
+  type: GraphQLNonNull(ArchiveTimelineEventPayload),
+  description: `Archive a timeline event`,
+  args: {
+    timelineEventId: {
```

Comment:
since this mutation affects multiple timeline events, I don't think we need this argument anymore.
We could also get rid of the `teamId` since we need to verify that `teamId` is attached to `meetingId`.

Reply:
looking at this with some fresh eyes, I think it makes sense to _only_ have the timelineEventId and to remove the `meetingId, teamId` args. That way we can use this same mutation for all sorts of timeline events, even those not linked to a meeting or team